lib/bolt/config.rb

```diff
@@ -48,7 +48,9 @@ class Config
       winrm: {
         'connect-timeout' => 10,
         'ssl' => true,
-        'ssl-verify' => true
+        'ssl-verify' => true,
+        'file-protocol' => 'winrm',
+        'smb-port' => 445
```

Comment:
@joshcooper / @adreyer I needed to add an `smb-port` configuration option for testing purposes. We had previously discussed the potential for multiple config options for SMB at https://github.com/puppetlabs/bolt/pull/856#discussion_r253982265 and where they should land in the options hierarchy.

If we intend to add more SMB options, is now the right time to break that out to a separate category?

Reply:
If we have defaults for these options putting them at the top level may be best. Otherwise we add complexity around nested merging that I don't think we're ready to handle yet. We can always migrate to a nested hash later if there are so many smb options that it becomes a problem.

Reply:
👍 	Thanks

I'd also prefer not to deal with adding nested structures and any resulting problems.